src/modules/ROOT/pages/changelog/202309/00.json

```diff
@@ -0,0 +1,261 @@
+      ]
+    },
+    {
+      "what": [
+        "unsaved changes warning message when modifying a record and trying to navigate to a different page."
+      ],
+      "why": [
+        "to prevent losing changes that haven't been saved yet."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1367"
+      ]
+    },
+    {
+      "what": [
+        "support for {PRODUCT_NAME} to automatically manage and change the underlying database when storing records."
+      ],
+      "why": [
+        "to simplify the process of using dedicated and customized storage locations for your records since manual intervention, for most cases, is no longer needed."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/bc923209f4745cf5ddfcd4987d926da5029670c6",
+        "https://github.com/cortezaproject/corteza/commit/0695eb71188c2d0c27b5186fd29120574a9a8534",
+        "https://github.com/cortezaproject/corteza/commit/4224ba9828c6d5bdc75aa618573fb99cac147062",
+        "https://github.com/cortezaproject/corteza/commit/b86f67cf6ff413912aa5dff6bd7f160f71b4de6e",
+        "https://github.com/cortezaproject/corteza/commit/1eb360f43e60bfbb149e6e4e64364c3ad1ad6b15",
+        "https://github.com/cortezaproject/corteza/commit/2bcf86ef3c1d28a2965b8d0947fa86c2b256cae8"
+      ]
+    },
+    {
+      "what": [
+        "an option to create a custom button for a record page in its page layout configurator that will be present when viewing the page layout."
+      ],
+      "why": [
+        "to provide the user with the option to create a custom button."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1484"
+      ]
+    },
+    {
+      "what": [
+        "an icon containing an exclamation mark to a page block toolbar indicating that the concrete page block is used in another page layout."
+      ],
+      "why": [
+        "to mark a page block that is already used in another page layout."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1495"
+      ]
+    },
+    {
+      "what": [
+        "the option to change the default colors of {PRODUCT_NAME} applications in {APP_NAME_ADMIN}."
+      ],
+      "why": [
+        "to provide the user with more freedom to style {PRODUCT_NAME}."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1469"
+      ]
+    },
+    {
+      "what": [
+        "a rounded circle icon that marks a configuration option as performance impacting."
+      ],
+      "why": [
+        "to mark which features would impact the performance of an application."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1490"
+      ]
+    }
+  ],
+
+  "changed": [
+    {
+      "what": [
+        "triggering of unsaved changes message after user wants to delete a modified resource like a page, namespace, module or chart."
```

Comment:
This would sound more like a bugfix than a change.
Also, if this change was made to a thing which was not yet released, we don't note it as a changelog entry -- rather add the refs of this entry to the original one.